in-cluster/default.yaml

```diff
@@ -128,6 +128,10 @@ spec:
         collectorName: rook-ceph-logs
         namespace: rook-ceph
         name: kots/rook
+    - logs:
```

Comment:
i think we should use a `selector` here so it only grabs the coredns logs and not everything in `kube-system` and then puts it under `logs/coredns` in the bundle

Reply:
Huh, good catch